mindjunkies/forums/views.py

```
@@ -55,89 +56,204 @@ def get_context_data(self, **kwargs):
+class ForumThreadDetailsView(LoginRequiredMixin,CourseContextMixin, TemplateView):
     template_name = "forums/forum_thread_details.html"
 
+    def get_module(self):
+        return get_object_or_404(
+            Module.objects.prefetch_related("forum_posts"),
+            id=self.kwargs.get("module_id"),
+        )
     def get_topic(self):
-        return get_object_or_404(ForumTopic, slug=self.kwargs.get("topic_slug"))
+        return get_object_or_404(ForumTopic, id=self.kwargs.get("topic_id"))
+   
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)